patches/chrome-browser-ui-webui-version_ui.cc.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/ui/webui/version_ui.cc b/chrome/browser/ui/webui/version_ui.cc
+index 715b6b037a97e4fe219bb802851fa92c4a086fd0..29ecee538750032165024e75dc87b23c6e6124c4 100644
+--- a/chrome/browser/ui/webui/version_ui.cc
++++ b/chrome/browser/ui/webui/version_ui.cc
+@@ -17,6 +17,7 @@
+ #include "chrome/common/url_constants.h"
+ #include "chrome/grit/chromium_strings.h"
+ #include "chrome/grit/generated_resources.h"
++#include "components/grit/brave_components_strings.h"
+ #include "components/grit/components_resources.h"
+ #include "components/strings/grit/components_strings.h"
+ #include "components/version_info/version_info.h"
+@@ -52,6 +53,8 @@ WebUIDataSource* CreateVersionUIDataSource() {
+       WebUIDataSource::Create(chrome::kChromeUIVersionHost);
+ 
+   // Localized and data strings.
++  html_source->AddLocalizedString(version_ui::kBraveRewardsDetails,
```

Comment:
we shouldn't be patching here for this. cc @petemill for a better place to put this

Reply:
actually I don't think this should be needed at all. Whatever version info we need can go in the internals ui page itself or in the rewards ui

Reply:
I'm totally happy to move this, but it was specifically requested to be here in https://github.com/brave/brave-browser/issues/1174#issuecomment-445906938.